modules/special-rules/pages/infiltrators.adoc

```diff
@@ -1,2 +1,18 @@
 = Infiltrators
 
+These units are the expert scouts and commandos who gather information, set ambushes, sabotage installations, and so on.
+
+---
+
+Apply these rules to {infiltrators} units:
```

Comment:
Should this be?
```suggestion
Apply these rules to units with the {infiltrators} ability:
```

Reply:
I want to tweak this a bit further, along with the text below - but I'll apply this suggestion for now to resolve this conversation.